### ISSUE TITLE
Clarify when "WS-Trust 1.3 Username/Mixed" is required

### ISSUE THREAD

**Issue by p0shkar** [MicrosoftDocs/memdocs#87]
"If using ADFS and the enrollment profile has Authenticate with Company Portal instead of Setup Assistant set to No, WS-Trust 1.3 Username/Mixed endpoint Learn more is required."

When I first read this I thought you meant that it is required when using Company Portal, but when I re-read it I think you mean when using Setup Assistant.

Maybe rephrase it like this:
"If using ADFS and the enrollment profile has Authenticate with Company Portal set to No, WS-Trust 1.3 Username/Mixed endpoint Learn more is required."
Since there is no option which is called "Company Portal instead of Setup Assistant".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bd130267-c54b-41e3-d534-831d4a86bae6
* Version Independent ID: bdfa7eca-a356-3dde-7643-9a759f952007
* Content: [Enroll iOS/iPadOS devices - Device Enrollment Program - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/enrollment/device-enrollment-program-enroll-ios#create-an-apple-enrollment-profile)
* Content Source: [memdocs/intune/enrollment/device-enrollment-program-enroll-ios.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/enrollment/device-enrollment-program-enroll-ios.md)
* Service: **microsoft-intune**
* Sub-service: **enrollment**
* Technology: ****
* GitHub Login: @ErikjeMS
* Microsoft Alias: **erikje**

**Comment by ErikjeMS** [comment]
Investigating with PM...
(6973948)

**Comment by ErikjeMS** [comment]
@p0shkar The UI has changed. I'll update the article today. Thanks!
